include/openssl/pkcs7.h

```diff
@@ -446,10 +446,12 @@ OPENSSL_EXPORT OPENSSL_DEPRECATED PKCS7 *PKCS7_sign(X509 *sign_cert,
 // Flags: If |PKCS7_NOVERIFY| is specified, trust chain validation is skipped.
 // This function also enforces the behavior of OpenSSL's |PKCS7_NO_DUAL_CONTENT|
 // meaning that |indata| may not be specified if |p7|'s signed data is attached.
+// OpenSSL will skip checking certs in |p7| if |PKCS7_NOINTERN| is set. While we
+// define that flag for build compatibility, we do not honor it.
```

Comment:
But didn't you change the logic so that we do honor it now?

Reply:
Ah, forgot to update this.